llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -2391,6 +2390,18 @@ void VPlanTransforms::createInterleaveGroups(
   }
 }
 
+void VPlanTransforms::dissolveLoopRegions(VPlan &Plan) {
+  // Replace loop regions with explicity CFG.
+  SmallVector<VPRegionBlock *> LoopRegions;
+  for (VPRegionBlock *R : VPBlockUtils::blocksOnly<VPRegionBlock>(
+           vp_depth_first_deep(Plan.getEntry()))) {
+    if (!R->isReplicator())
+      LoopRegions.push_back(R);
+  }
+  for (VPRegionBlock *R : LoopRegions)
+    R->removeRegion();
```

Comment:
Perhaps clearer to rename into something like
```suggestion
    R->dissolveToCFGLoop();
```
to emphasize this is not "just removing".